src/lib/utils.py

```diff
@@ -49,7 +49,7 @@ def pivot_table_date_columns(
 def table_rename(
     data: DataFrame,
     column_adapter: Dict[str, str],
-    remove_regex: str = r"[^a-z\s]",
+    remove_regex: str = r"[^a-z1-9\s]",
```

Comment:
I agree this was a bad default and your proposed change is better. Two questions:
1. Have you looked for all usages of `table_rename` to make sure this wouldn't break something?
2. Why 1-9 and not 0-9?

Reply:
Yeah, the default caused a bug that took me a couple of hours to find (because the table names clashed once the numbers were removed). I've added the zero, looking through the other usages now. I can't think of a reason why this wouldn't be backward incompatible. The only use where it could've been a problem is if an intentional collapse of two column names with numbers in them to have the same column name (which can then be accessed via loc)

Reply:
I was hit by exactly the same bug, so I appreciate you took the initiative to fix it for good :-)

And I agree, the only reason why this would break something is if a data source relies on numbers being removed from the column names which seems unlikely. I have tried to always use the full, original column name every time I call `table_rename` so auditing this should be straightforward.

Reply:
I've looked through all occurrences. I note that there are other places where the default regex was substituted with the one I'm proposing as the new default, so if you remain worried that this is a breaking change, I won't change the default. That said, my investigation did not reveal any breaking cases, some standouts below,

- us_de_authority.py has numbers in the column name, but the regex arg was changed. 
- us_nyc_authority.py currently would remove the `prefix` number from the column. I would've thought this would lead to a bug, but luckily it does not because each dataframe is for the specific prefix. Still, I think the behaviour is better with the new default in this case. 
- `us_wa_authority.py` already passes in the identical regex to the new default. 
- `usasleep.py` will now use the numbers in e(0), STATE2KX and CNTY2KX instead of removing them, but this should not cause any issue during the rename.
- `es_authority.py` has "Unnamed: 0" in the column adaptor, which after regex removal would have previously become `Unnamed`. Similar for `Dosis entregadas (1)` and `Dosis administradas(2)`. The new names would include the numbers i.e. Unnamed0, Dosisentregadas1 and Dosisadministradas2. I don't think this would be an issue.
- `fr_authority.py` would have had a few bugs under the old default, but a different regex is passed anyway.